src/main/java/com/sprint/mission/discodeit/JavaApplication.java

```diff
@@ -3,100 +3,46 @@
 import com.sprint.mission.discodeit.entity.Channel;
 import com.sprint.mission.discodeit.entity.Message;
 import com.sprint.mission.discodeit.entity.User;
-import com.sprint.mission.discodeit.service.jcf.JCF_channel;
-import com.sprint.mission.discodeit.service.jcf.JCF_message;
-import com.sprint.mission.discodeit.service.jcf.JCF_user;
 
+
+import com.sprint.mission.discodeit.service.file.FileChannel;
+import com.sprint.mission.discodeit.service.file.FileMessage;
+import com.sprint.mission.discodeit.service.file.FileUser;
 import java.util.List;
-import java.util.Scanner;
 import java.util.UUID;
 
+// 해야할 일
+// User와 Channel의 연결하는 중간 모델 적용
+// 모델 일대다 HashMap으로 교체
+// 싱글톤 구현
 
 public class JavaApplication {
 
     public static void main(String[] args) {
+        FileUser fileUser = new FileUser();
+        FileMessage fileMessage = new FileMessage();
+        FileChannel fileChannel = new FileChannel();
 
-        JCF_channel jcf_channel = new JCF_channel();
-        JCF_user jcf_user = new JCF_user();
-        JCF_message jcf_message = new JCF_message();
-
-
-        System.out.println("유저 생성");
-        jcf_user.creat(new User("user1"));
-        jcf_user.creat(new User("user2"));
-        jcf_user.creat(new User("user3"));
-        jcf_user.creat(new User("user4"));
-        List<User> userList = jcf_user.allWrite();
-        for (User item : userList) {
-            System.out.println("- " + item);
-        }
-        System.out.println("\n");
-
-        System.out.println("채널 생성");
-        jcf_channel.creat(new Channel("channel1"));
-        jcf_channel.creat(new Channel("channel2"));
-        jcf_channel.creat(new Channel("channel3"));
-        jcf_channel.creat(new Channel("channel4"));
-        List<Channel> channelList = jcf_channel.allWrite();
-        for (Channel item : channelList) {
-            System.out.println("- " + item);
-        }
-        System.out.println("\n");
-
-        System.out.println("채널에 채팅 생성");
-        UUID user1 = jcf_user.write("user1");
-        UUID user2 = jcf_user.write("user2");
-        UUID user3 = jcf_user.write("user3");
-        UUID channel = jcf_channel.write("channel1");
-        String userName1 = jcf_user.getName(user1);
-        String userName2 = jcf_user.getName(user2);
-        String userName3 = jcf_user.getName(user3);
-        jcf_message.creat(new Message(user1, "Message Test1!!", channel, userName1));
-        jcf_message.creat(new Message(user2, "Message Test2!!", channel, userName2));
-        jcf_message.creat(new Message(user3, "Message Test3!!", channel, userName3));
 
-        List<Message> messageList = jcf_message.getMessage(channel);
-        for (Message item : messageList) {
-            System.out.println("- " + item);
+        fileUser.creat("admin");
+        fileUser.creat("admin1");
+        fileUser.creat("admin2");
+        UUID user1 = fileUser.findByName("admin");
+        List<User> userId = fileUser.findByAll();
```

Comment:
users가 좀 더 정확할 것 같아요.
```suggestion
        List<User> users = fileUser.findByAll();
```